hack/cherry_pick_pull.sh

```diff
@@ -128,7 +133,8 @@ trap return_to_kansas EXIT
 
 SUBJECTS=()
 function make-a-pr() {
-  local rel="$(basename "${BRANCH}")"
+  local rel
+  rel="$(basename "${BRANCH}")"
```

Comment:
change reason:
```
In hack/cherry_pick_pull.sh line 131:
  local rel="$(basename "${BRANCH}")"
        ^-- SC2155: Declare and assign separately to avoid masking return values.
```